test_scripts/API/OnSeekMediaClockTimer/OnSeekMediaClockTimer_With_Fake_Parameter_In_seekTime_Structure.lua

```diff
@@ -0,0 +1,103 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [HMI RPC validation]: SDL behavior: HMI sends request (response, notification) with fake parameters that SDL should transfer to mobile
```

Comment:
@OVikhrov, please update Requirement summary also with related to OnSeekMediaClockTimer functional requirement